cmd/cli/plugin/pinniped-auth/login.go

```diff
@@ -91,6 +92,7 @@ func loginOIDCCommand(
 			fmt.Sprintf("--skip-browser=%s", strconv.FormatBool(loginOptions.skipBrowser)),
 			fmt.Sprintf("--session-cache=%s", loginOptions.sessionCachePath),
 			fmt.Sprintf("--debug-session-cache=%s", strconv.FormatBool(loginOptions.debugSessionCache)),
+			fmt.Sprintf("--credential-cache=%s", loginOptions.credentialCachePath),
```

Comment:
I think we only want this flag to get passed to the Pinniped CLI if we are utilizing a later version of Pinniped, so we could use `conciergeIsClusterScoped` to detect that.  Otherwise, we'll likely get an error from Pinniped CLI.

Reply:
Ah, yup yup, lemme fix that also.